packages/file-type-icons/src/version.ts

```diff
@@ -0,0 +1,3 @@
+// @uifabric/file-type-icons@6.1.1 - Do not modify this file, the file is generated as part of publish but not checked in
```

Comment:
>but not checked in [](start = 103, length = 18)

If it is not checked in, why is it being checked in atm? I'm confused

Reply:
It is just a dummy file that is checked in here, but the build process will replace it in flight, but not checked in ... hmmm I'll word smith this a bit more.